metrictank.go

```diff
@@ -175,6 +176,9 @@ func main() {
 	case 6:
 		log.Level(log.FATAL)
 	}
+	mdata.LogLevel = logLevel
+	inKafkaMdm.LogLevel = logLevel
+	api.LogLevel = logLevel
```

Comment:
why did you move the assignments down instead of just keeping them as is?

Reply:
Honestly, a testing artifact. I did think that since we're initializing the logger first, we'd want to configure it first.